src/main/java/com/work/management/converter/ListConverter.java

```diff
@@ -0,0 +1,22 @@
+package com.work.management.converter;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.util.Arrays;
+import java.util.List;
+
+@Converter
+public class ListConverter implements AttributeConverter<List<String>,String> {
+    private static final String SPLIT_CHAR = ";";
+
+    @Override
+    public String convertToDatabaseColumn(List<String> list) {
+
+        return String.join(SPLIT_CHAR,list);
+    }
+
+    @Override
+    public List<String> convertToEntityAttribute(String string) {
+        return Arrays.asList(string.split(SPLIT_CHAR));
+    }
```

Comment:
always return immutable list from functions... use ImmutableList of Gauva here..https://guava.dev/releases/20.0/api/docs/com/google/common/collect/ImmutableList.html...

This method should be :
public `ImmutableList<String>`convertToEntityAttribute(String message){
return ImmutableList.of(message.split(DELIMITER));
}